docker/deb-rpm-s3/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+#
+# Build script for the deb-rpm-s3 docker container.
+#
+
+cd "$(dirname "$0")"
+
+if [ ! -e "elasticsearch.asc" ]; then
+    cat << EOF
+You must place a copy of the Elasticsearch GPG signing key (named
+elasticsearch.asc) into
```

Comment:
instead of adding key to image, how about 'downloading' it in running container? If container is executed with '--rm' what's the risk of pushing it by accident?